### PR TITLE
Incorrect documentation around password policies

### LINKED ISSUE #19363: Incorrect documentation around password policies
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

docs

### Describe the bug

Latest docs for password policies https://www.keycloak.org/docs/latest/server_admin/index.html#_password-policies mentions this:
```
After saving the policy, Keycloak enforces the policy for new users and sets an Update Password action for existing users to ensure they change their password the next time they log in. 
```

This is not correct as Keycloak does NOT enforce existing users to update their password. For example assume that administrator adds new password policy `Minimum length 8` and there is already user `john` with password `foo`.

During the subsequent logins after password policy change, the user is able to login with this password and he is *not* forced to update it during login. However when he wants to update his password (for example in account console), he *is*enforced to use password according to password policy and hence his password must be 8 chars or longer. This also applies for all newly registered users (after password policy was created).

So what we can do is:
- Update docs and clarify this (for example add warning that admins should set their password policy from the beginning of the realm and/or add "Update password" to existing users and/or use "Expire password" to make sure that users update their passwords in next N days, which will in fact adjust to new password policies)
- Implement something to make sure that existing users are forced to update passwords as well. There is related issue #14150

### Version

21.0.1

### Expected behavior

a

### Actual behavior

a

### How to Reproduce?

a

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by pedro-hos**
closes #19363

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
@pedro-hos Thanks! This looks good to me assuming that we go in a way of handling this in the documentation. Will discuss this within the team as well.

File: docs/documentation/server_admin/topics/authentication/password-policies.adoc [hunk 1]
```diff
@@ -15,10 +15,12 @@ When {project_name} creates a realm, it does not associate password policies wit
...
 image:images/password-policy.png[Password Policy]
 
-After saving the policy, {project_name} enforces the policy for new users and sets an Update Password action for existing users to ensure they change their password the next time they log in. For example:
+After saving the policy, {project_name} enforces the policy for new users.
 
-.Failed password policy
-image:images/failed-password-policy.png[Failed Password Policy]
+[NOTE]
+====
+The new policy will not be effective for existing users. So, Make sure that you set the password policy from the beginning of the realm creation and/or add "Update password" to existing users and/or use "Expire password" to make sure that users update their passwords in next "N" days, which will in fact adjust to new password policies.
```
Comment (andymunro) [inline docs/documentation/server_admin/topics/authentication/password-policies.adoc:22]:
```suggestion
The new policy will not be effective for existing users. Therefore, make sure that you set the password policy from the beginning of the realm creation or add "Update password" to existing users  or use "Expire password" to make sure that users update their passwords in next "N" days, which will actually adjust to new password policies.
```

**Review by andymunro — CHANGES_REQUESTED** [review]
@mposolda  A few suggestions.

**Comment by mposolda** [comment]
@andymunro Thanks!

**Review by mposolda — APPROVED** [review]

**Comment by pedro-hos** [comment]
Thank you @andymunro and @mposolda

**Pull request merged**